### PR TITLE
Add volume for important data

### PR DESCRIPTION
**Pull request by jsf9k**
This volume gets mounted at `/opt/PCA`.

### CODE REVIEW

File: terraform/teamserver_ec2.tf [hunk 1]
```diff
@@ -40,3 +40,47 @@ resource "aws_eip_association" "eip_assoc" {
...
+resource "aws_volume_attachment" "teamserver_data_attachment" {
+  device_name = "/dev/xvdb"
+  volume_id = "${aws_ebs_volume.teamserver_data.id}"
+  instance_id = "${aws_instance.teamserver.id}"
+
+  # Terraform attempts to destroy the volume attachments before it
+  # attempts to destroy the EC2 instance they are attached to.  EC2
+  # does not like that and it results in the failed destruction of the
+  # volume attachments.  To get around this, we explicitly terminate
+  # the cyhy_reporter volume via the AWS CLI in a destroy provisioner;
```
Comment (dav3r) [inline terraform/teamserver_ec2.tf]: `cyhy_reporter` is mentioned in this comment- you probably don't want that.

File: terraform/teamserver_ec2.tf [hunk 2]
```diff
@@ -40,3 +40,47 @@ resource "aws_eip_association" "eip_assoc" {
...
+  # this gracefully shuts down the instance and allows terraform to
+  # successfully destroy the volume attachments.
+  provisioner "local-exec" {
+    when = "destroy"
+    command = "aws --region=${var.aws_region} ec2 terminate-instances --instance-ids ${aws_instance.teamserver.id}"
+    on_failure = "continue"
+  }
+
+  # Wait until cyhy_reporter instance is terminated before continuing on
```
Comment (dav3r) [inline terraform/teamserver_ec2.tf]: Also here is another `cyhy_reporter` mention.

**Comment by jsf9k** [comment]
@dav3r - My latest commit should address your astute comments on my comments.

**Review by dav3r — APPROVED** [review]
💪 work @jsf9k

**Pull request merged**